iaso/api/forms.py

```diff
@@ -139,6 +143,10 @@ def get_latest_form_version(obj: Form):
     def get_org_unit_types(obj: Form):
         return [t.as_dict() for t in obj.org_unit_types.all()]
 
+    @staticmethod
+    def get_reference_form_of_org_unit_types(obj: Form):
+        return [org_unit.as_dict() for org_unit in obj.reference_of_org_unit_types.all()]
+
```

Comment:
Looks to me that these are not org_unit but reference forms (the name org_unit is wrong?)

Reply:
It looks a bit strange when seen out of context, but it's right.

I'm adding this method to `FormSerializer` to populate `reference_form_of_org_unit_types`:

```python
serialized_form = FormSerializer(form)
serialized_form.reference_form_of_org_unit_types
```

So it returns `org_unit`s, [as asked by @bmonjoie ](https://bluesquare.atlassian.net/browse/IA-2411?focusedCommentId=59205).

Can't think of a better name…